### DISCUSSION TITLE
Way to stop communication with note from python

### DISCUSSION THREAD

**Discussion by gohilurvish** [RhetTbull/macnotesapp#44] [General]
First of all, thank you so much for this effort. This library is easy to use.

I'm trying to make a python script which read a note and parse some data.
First step is to list notes title and ask user to select.
Once done with selection, the script will read note body.
At this point, the script no more needs to communicate with Notes.app. Though script has a lot of work to do which needs user input but no comms with Notes.app.

I see that a when communication with Notes.app starts, a python rocket icon appears in the mac dock. This doesn't go away until python script ends.

What I am looking for is that rocket icon/communication with Notes.app to stop once note body is read and not stay until end of scripts which may be minutes away.

I tried .quit() but that is not working as expected.

**Comment by RhetTbull** [comment]
I'm glad the library is useful to you! The "rocket icon" is launched when Python is running interactively. On macOS if Python interacts with the GUI, the icon appears and once it appears, I don't think you can make it disappear. This isn't directly related to Notes but due to the fact that macnotesapp is interacting with macOS. You used to be able to edit the Info.plist in the Python app to make it not show the icon but I'm not sure this is still possible with System Integrity Protection (SIP) but I've not tried it.

**Reply by gohilurvish** [reply]
Thank you for the quick reply.
In that case I'd call another script to get the work done so it leaves the UI sooner.
